build.psm1

```diff
@@ -558,6 +558,20 @@ Fix steps:
         }
     }
 
+    if ($Environment.IsWindows) {
+        ## need RCEdit to modify the binaries embedded resources
+        if (-not (Test-Path "~/.rcedit/rcedit-x64.exe"))
+        {
+            throw "RCEdit is required to modify pwsh.exe resources, please run 'Start-PSBootStrap' to install"
+        }
+
+        $ProductVersion = (git --git-dir="$PSScriptRoot/.git" describe) -Replace '^v'
```

Comment:
Use release tag parameter to get version if it was passed it.  This is needed for release builds.